pkg/virt-controller/services/multus_annotations.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+type multusNetworkAnnotation struct {
+	InterfaceName string `json:"interface"`
+	Mac           string `json:"mac,omitempty"`
+	NetworkName   string `json:"name"`
+	Namespace     string `json:"namespace"`
+}
+
+func generateMultusCNIAnnotation(vmi *v1.VirtualMachineInstance) (string, error) {
+	var multusNetworkAnnotationPool []multusNetworkAnnotation
+
+	multusNonDefaultNetworks := filterMultusNonDefaultNetworks(vmi.Spec.Networks)
+	for i, network := range multusNonDefaultNetworks {
+		serializedInterface := newMultusAnnotationData(vmi, network, fmt.Sprintf("net%d", i+1))
```

Comment:
Just wandering why you call a return struct value "serialized". I think that "serialized" refers to something that can be send over to a remote destination through a protocol, like a string, json, xml. Am I getting confused here?

If I am not mistaken, the serialization occurs when `json.Marshal` is called, right?

Reply:
This was addressed already in https://github.com/kubevirt/kubevirt/compare/8d1b97ee6a5652fefea643b567d60046efb7c7fe..c1709d426e838e63a95baffc30a5a3b6a907f610

Reply:
The link does not work, can you update it?

Reply:
Works for me. Nothing I can do I'm afraid.

But, at the end of the day, it's been addressed; resolving.